rpm/droid-system-f5121.spec

```diff
@@ -59,6 +59,24 @@ mkdir tmp
 os.execute("rm -rf /system/vendor/firmware")
 os.execute("rm -rf /system/vendor/lib/egl")
 os.execute("rm -rf /system/vendor/lib64/egl")
+os.execute("rm -rf /system/app")
+os.execute("rm -rf /system/etc/security/otacerts.zip")
+os.execute("rm -rf /system/fonts")
+os.execute("rm -rf /system/framework")
+os.execute("rm -rf /system/lib/libpac.so")
+os.execute("rm -rf /system/lib/libart.so")
+os.execute("rm -rf /system/lib/libart-compiler.so")
+os.execute("rm -rf /system/lib/libpdfium.so")
+os.execute("rm -rf /system/lib64/libpac.so")
+os.execute("rm -rf /system/lib64/libart.so")
+os.execute("rm -rf /system/lib64/libart-compiler.so")
+os.execute("rm -rf /system/lib64/libpdfium.so")
+os.execute("rm -rf /system/lib/modules")
+os.execute("rm -rf /system/media")
+os.execute("rm -rf /system/priv-app")
+os.execute("rm -rf /system/tts")
+os.execute("rm -rf /system/usr/keychars")
+os.execute("rm -rf /system/usr/keylayout")
```

Comment:
This is wrong, if we do this here it means that it is still shipped with the package and removed during installation. We should remove these files already in packaging i.e. %install phase. The old removals that were here was because of the architecture change.